docs/AzurePipelines.md

```diff
@@ -78,7 +78,7 @@ Add the following task to your Azure Pipeline:
 
 You'll need to configure an Azure service principal as a [service connection](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=vsts) and set the name of the service connection to the `azureSubscription` variable.
 
-Also be sure to set the [`azureSubscriptionId`](LuisEndpointConfiguration.md#azuresubscriptionid), [`azureResourceGroup`](LuisEndpointConfiguration.md#azureresourcegroup), [`azureLuisResourceName`](LuisEndpointConfiguration.md#azureluisresourcename), [`luisEndpointKey`](LuisEndpointConfiguration.md#luisendpointkey) and [`luisEndpointRegion`](LuisEndpointConfiguration.md#luisendpointregion) variables.
+Also be sure to set the [`azureSubscriptionId`](LuisEndpointConfiguration.md#azuresubscriptionid), [`azureResourceGroup`](LuisEndpointConfiguration.md#azureresourcegroup), [`luisPredictionResourceName`](LuisEndpointConfiguration.md#luisPredictionResourceName), and [`luisEndpointKey`](LuisEndpointConfiguration.md#luisendpointkey).
```

Comment:
Just a suggestion: Since you're adopting 'luisPredictionResourceName', I wonder if you should consider changing 'luisEndpointKey' to 'luisPredictionKey'.   Both Authoring and Prediction resources have endopoints so calling it the luisEndpointKey could be considered ambiguous.
Also, the LUIS PG have traditionally called this the 'Subscription Key' but I believe that they too are moving over to 'Prediction Key'